README.md

```diff
@@ -20,7 +20,13 @@ is the recommended way of installing Alertmanager.
 
 ### Docker images
 
-Docker images are available on [Quay.io](https://quay.io/repository/prometheus/alertmanager).
+Docker images are available on [Quay.io](https://quay.io/repository/prometheus/alertmanager) or [Docker Hub](https://hub.docker.com/r/prom/alertmanager/).
+
+You can launch an Alertmanager container for trying it out with
+
+    $ docker run --name alertmanager -d -p 127.0.0.1:9093:9093 prom/alertmanager
```

Comment:
Given the new rate limiting policies on Docker hub, can you change it to use quay.io/prometheus/alertmanager instead?